docs/Tutorials/HelloWorld/HelloWorld.md

```diff
@@ -104,21 +107,23 @@ To build this component run `fprime-util build` in the current folder.
 A component model defines the interface of the component with the rest of the F´ system and with the ground system F´
 communicates with. In this case we intend to define a command, an event, and a telemetry channel as specified above.
 
-Open the model file `HelloWorld.fpp` and add replace the line
+Open the model file `HelloWorld.fpp` and add replace the line:
 
-`async command TODO opcode 0`
+```
+async command TODO opcode 0
+```
 
 with the following:
 
 ```
 @ Command to issue greeting with maximum length of 20 characters
 async command SAY_HELLO(
-    greeting: string size 20 @ Greeting to repeat in the Hello event
+    greeting: string size 20 @< Greeting to repeat in the Hello event
```

Comment:
This was already updated in `devel` in https://github.com/nasa/fprime/pull/1985/files so I'm confused as to why it still shows up here (I understand you probably branched off earlier than the change). Could that cause a merge conflict?

Reply:
Apparently no conflict.